CardPayments/src/main/java/com/paypal/android/cardpayments/CardClient.kt

```diff
@@ -150,6 +151,10 @@ class CardClient internal constructor(
 
             is CardStatus.ApproveOrderSuccess -> notifyApproveOrderSuccess(status.result)
             is CardStatus.ApproveOrderCanceled -> notifyApproveOrderCanceled(status.orderId)
+            is CardStatus.UnknownError -> {
+                Log.d("PayPalSDK", "An unknown error occurred: ${status.error.message}")
```

Comment:
Currently, it's unclear which listener should receive this error message.

This should only occur when the `authState` variable sent to the client is tampered with. Once listeners are removed from the API, this will no longer be an issue.